src/main/nav_create_bundle.py

```
@@ -0,0 +1,261 @@
+
+from pds4.bundle_data import generate_bundle_data_files
+from pds4.collections import generate_collection_files, generate_global_index_files
+
+
+DATASET: DataSet | None = None
+DATASET_NAME: str | None = None
+MAIN_LOGGER: pdslogger.PdsLogger | None = None
+
+
+def add_common_arguments(parser: argparse.ArgumentParser,
+                         for_labels: bool = False) -> None:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)